src/main/drivers/max7456.c

```diff
@@ -68,6 +68,9 @@ static uint8_t spiBuff[MAX_CHARS2UPDATE*6];
 
 static uint8_t  videoSignalCfg   = 0;
 static uint8_t  videoSignalReg   = VIDEO_MODE_PAL | OSD_ENABLE; //PAL by default
+static uint8_t  hosValue;
```

Comment:
`hosValue` and `vosValue` are not the offset themselves, but encoded and ready to write to register (HOS and VOS registers respectively) values. I will rename them to emphasize this and add some comments.